helper_functions/confirmOtp.js

```diff
@@ -0,0 +1,24 @@
+const Otp = require('../models/Otp');
+
+const confirmOtp = async (email, otp) => {
+  let otp_object;
+
+  //finding otp with given fields
+
+  try {
+    otp_object = await Otp.findOne({
+      email,
+      text: otp,
+    });
+  } catch (e) {
+    throw new Error(e);
+  }
+
```

Comment:
Dont use try catch here, use in the outer function

Reply:
@impiyush83 can you elaborate?

Reply:
@XshubhamX  Its no need to put try-catch in the child/helper function. You just throw an error from helper functions and let the handler function/function who is calling this confirmOtp have the try-catch block.

Reply:
ok got it